build-tools/xa-prep-tasks/Xamarin.Android.BuildTools.PrepTasks/ProcessLogcatTiming.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text.RegularExpressions;
+
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+namespace Xamarin.Android.BuildTools.PrepTasks
+{
+	public class ProcessLogcatTiming : Task
+	{
+		public string Logcat { get; set; }
```

Comment:
This should be `[Required]`, and should imply that it's a filename. I'd suggest calling it `AdbLogcatFilename`?